src/jsdom.js

```diff
@@ -29,7 +29,8 @@ function fromSource(src, options) {
             FetchExternalResources: ['script'],
             ProcessExternalResources: ['script']
         },
-        virtualConsole: jsdom.createVirtualConsole().sendTo(console)
+        virtualConsole: jsdom.createVirtualConsole().sendTo(console),
+        created: options.jsdomCreated
```

Comment:
nvm, I went ahead and did it myself in #324. I wonder if I should cherry pick this change in that branch too.